triage-process.md

```diff
@@ -0,0 +1,59 @@
+# High Level Proposal:
+1. Define and publish a process to handle issues in a structured manner.
+    1. Issue Handling Process
+    1. Issue Stages
+    1. Roles
+    1. SLA
+
+## Process Definition
+
+### Issue Handling Process
+1. All incoming issues should be assessed. These are items that a Triager would look at on a daily basis
+1. Every issue triaged should be assessed for its workability. If assessing the workability of the issue requires more information, the Triager would collaborate with the OP until workability is determined.
+    1. Workability Classification (also below are label names):
+        1. feature-request
+        1. enhancement
+        1. bug
+        1. wontfix
+        1. docs
+1. Workable Issues should be prioritized and Labeled as one of 3 priorities
+    * P0 = Critical (Drop everything and work on it.)
+    * P1 = Important (Work on it if no P0 are present. Use judgment to prioritize items within a list of P1s)
+    * P2 = Low (Work on it when no P1 are left or some one from community wants to contribute)
+1. Prioritized and workablity classified issues  issues should are considered as ready-for-work. This should signal to any contributor willing to help that they can work on this issue.
+1. Once a contributor is working on this issue, the issue should be assigned to them. (Assigned issues are considered to be in in-progress stage.)
+1. Once work is complete and the issue has been resolved, simply close the Issue. A closed issue with associated merged PR is considered as completed. 
```

Comment:
Issue(s) should be capitalized (or not) consistently.

Reply:
done. I went thru the full doc so we should be good but PTAL incase I have missed any instance.